src/anomalib/pipelines/benchmark/pipeline.py

```diff
@@ -0,0 +1,44 @@
+"""Benchmarking."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+from jsonargparse import ArgumentParser, Namespace
+
+from anomalib.pipelines.components.base import Pipeline, Runner
+from anomalib.pipelines.components.runners import ParallelRunner, SerialRunner
+
+from .generator import BenchmarkJobGenerator
+
+
+class Benchmark(Pipeline):
+    """Benchmarking orchestrator."""
```

Comment:
Are we still using `orchestrator`? Or is it a left-over?

Reply:
I missed this